source/includes/steps-enable-authentication-in-shardcluster-nodowntime-auth-mongos.yaml

```diff
@@ -0,0 +1,55 @@
+stepnum: 1
+level: 4
+title: "Remove ``transitionToAuth`` from the ``mongos`` configuration file"
+ref: modify-config-files
+pre: |
+
+  Remove the :setting:`security.transitionToAuth` key and its value from the
+  configuration file created during this tutorial.
+  
+  .. code-block:: yaml
+
+     security:
+        keyFile: <path-to-keyfile>
+
+---
+stepnum: 2
+level: 4
+ref: restart-mongos
+title: "Restart the ``mongos`` with the updated configuration file."
+pre: |
+
+  Connect to the :program:`mongos` and use the :method:`db.shutdownServer()`
+  method against the ``admin`` database to safely shut down the
+  :program:`mongos`.
+  
+  Restart one :program:`mongos` at a time to ensure that clients
+  can connect to the sharded cluster. If your cluster has only one
+  :program:`mongos`, this step results in downtime while the :program:`mongos`
+  is offline.
+  
+  .. code-block:: javascript
+  
+     db.getSiblingDB("admin").shutdownServer()
+  
+  You can then start the :program:`mongos` specifying the updated
+  configuration file, specifying the path to the config file using
+  :option:`--config`.. This configuration file no longer contains the
+  :setting:`security.transitionToAuth` setting, but retains the
+  :setting:`security.keyFile` setting.
```

Comment:
This sentence strikes me as odd since in step one, you made me remove it.  and then in step two, you mention the state of the file.